services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall:
```

Comment:
are u sure about type annotation for `all `method ?

Reply:
QuerySet[CinemaHall] ?